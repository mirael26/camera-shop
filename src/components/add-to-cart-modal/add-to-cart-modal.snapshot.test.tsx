import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import { productMock } from '../../test/mocks';
import AddToCartModal from './add-to-cart-modal';

jest.mock('./add-to-cart-modal-card/add-to-cart-modal-card', () => 'AddToCartModalCard');
jest.mock('./add-to-cart-modal-success/add-to-cart-modal-success', () => 'AddToCartModalSuccess');

test('AddToCartModal renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<AddToCartModal/>, { initialState: {
    cart: {
      addedToCartItem: productMock,
    }
  }});
  expect(asFragment()).toMatchSnapshot();
});
