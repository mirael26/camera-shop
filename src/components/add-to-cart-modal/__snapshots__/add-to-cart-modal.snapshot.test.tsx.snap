// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToCartModal renders correctly 1`] = `
<DocumentFragment>
  <div
    class="modal__content"
  >
    <addtocartmodalcard />
  </div>
</DocumentFragment>
`;
